actionview/lib/action_view/rendering.rb

```diff
@@ -35,24 +35,36 @@ def process(*) #:nodoc:
     end
 
     module ClassMethods
-      def view_context_class
-        @view_context_class ||= begin
-          supports_path = supports_path?
-          routes  = respond_to?(:_routes)  && _routes
-          helpers = respond_to?(:_helpers) && _helpers
-
-          Class.new(ActionView::Base) do
-            if routes
-              include routes.url_helpers(supports_path)
-              include routes.mounted_helpers
-            end
-
-            if helpers
-              include helpers
-            end
+      def _routes
+      end
+
+      def _helpers
+      end
+
+      def build_av_class(klass, supports_path, routes, helpers)
```

Comment:
can we rename this method? `av` is too much lingo. Maybe `build_view_context_class`?

Reply:
Yep.  That's a much better name. 3df7b96e38837c4880212def203c4d4fb359d99d